tfhe/src/shortint/engine/mod.rs

```diff
@@ -18,7 +18,7 @@ use std::fmt::Debug;
 mod client_side;
 mod public_side;
 mod server_side;
-mod wopbs;
+pub(crate) mod wopbs;
```

Comment:
It's needed to import `WopbsKeyCreationError`

I'll move `WopbsKeyCreationError` and revert this change

Reply:
not necessarily, I don't recall the pattern for Error declaration, if it makes sense like this it's fine, otherwise if we declare errors in the same spot then maybe we should prefer that, not sure